src/roi/Roi.ts

```diff
@@ -91,4 +138,516 @@ export class Roi {
   public getBorderPoints(options?: GetBorderPointsOptions): Array<Point> {
     return getBorderPoints(this, options);
   }
+
+  _computeBorderIDs(): void {
+    let borders = getBorders(this);
+    this.computed.borderIDs = [...borders.ids];
```

Comment:
I guess we could keep borders.ids and we don't need to create a new array here

In order to prevent to use this method we can do hte same as with `getBorders` and call `computeBorderIDs(this)`.